.env

```diff
@@ -38,7 +38,7 @@ REACT_APP_DOMAIN_REGEX_LOCAL="^(:?localhost:\d{2,5}|(?:127|192)(?:\.[0-9]{1,3}){
 REACT_APP_DOMAIN_REGEX_PR="^pr\d+--cowswap\.review"
 REACT_APP_DOMAIN_REGEX_DEV="^cowswap\.dev"
 REACT_APP_DOMAIN_REGEX_STAGING="^cowswap\.staging"
-REACT_APP_DOMAIN_REGEX_PROD="^cowswap\.exchange$"
+REACT_APP_DOMAIN_REGEX_PROD="(^cowswap\.exchange$) | (^swap\.cow\.fi$)"
```

Comment:
I would be surprised if the ` ` was allowed:
```suggestion
REACT_APP_DOMAIN_REGEX_PROD="(^cowswap\.exchange$)|(^swap\.cow\.fi$)"
```

<img width="529" alt="image" src="https://user-images.githubusercontent.com/4210206/174291424-0fd8887a-dd58-4052-b4c8-3bc40b816660.png">